services/net/filemonitor/Config/FilemonitorOptions.cs

```diff
@@ -17,5 +17,10 @@ public class FileMonitorOptions : IngestServiceOptions
     /// get/set - The path to private key files
     /// </summary>
     public string PrivateKeysPath { get; set; } = "";
+
+    /// <summary>
+    /// get/set - The text to insert into the body of a story that failed to be imported because the source file format was invalid.
+    /// </summary>
+    public string FailedStoryBody { get; set; } = "";
```

Comment:
We can control what is displayed to editors with a failed story.  It's very likely we're missing more than one story, but we have no way to tell presently.